public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -346,33 +358,41 @@ export class PanelInspector extends PureComponent<Props, State> {
   };
 
   render() {
-    const { last, tab, drawerWidth } = this.state;
+    const { panel, dashboard } = this.props;
+    const { loading, last, tab, drawerWidth } = this.state;
     const styles = getStyles();
     const error = last?.error;
 
     return (
       <Drawer title={this.drawerHeader} width={drawerWidth} onClose={this.onDismiss}>
-        <TabContent className={styles.tabContent}>
-          {tab === InspectTab.Data && this.renderDataTab()}
-          <CustomScrollbar autoHeightMin="100%">
-            {tab === InspectTab.Meta && this.renderMetadataInspector()}
-            {tab === InspectTab.Request && this.renderRequestTab()}
-            {tab === InspectTab.Error && this.renderErrorTab(error)}
-            {tab === InspectTab.Stats && this.renderStatsTab()}
-            {tab === InspectTab.PanelJson && this.renderJsonModelTab()}
-          </CustomScrollbar>
-        </TabContent>
+        {loading ? (
+          <div>loading...</div>
```

Comment:
Maybe add a spinner here?
```suggestion
          <div>Loading <Icon name="spinner" className=fa-spin" />
```